submission_runner.py

```diff
@@ -66,8 +60,11 @@
 flags.DEFINE_string(
     'data_dir',
     '~/',
-    'Dataset location'
-)
+    'Dataset location')
+flags.DEFINE_boolean(
+    'use_jax_rng',
+    False,
+    'Whether to use the Jax or Numpy RNG library.')
```

Comment:
Can you just add a 'For Pytorch, use the Numpy RNG library' to the description here to motivate the differences for users?

Reply:
done!